include/pinocchio/bindings/python/spatial/inertia.hpp

```diff
@@ -293,6 +310,260 @@ namespace pinocchio
+          .def(
+            "FromDynamicParameters", &PseudoInertia::template FromDynamicParameters<VectorXs>,
+            bp::args("dynamic_parameters"),
+            "Builds a pseudo inertia matrix from a vector of dynamic parameters."
+            "\nThe parameters are given as dynamic_parameters = [m, h_x, h_y, h_z, I_{xx}, "
+            "I_{xy}, I_{yy}, I_{xz}, I_{yz}, I_{zz}]^T.")
+          .staticmethod("FromDynamicParameters")
+
+          .def(
+            "FromMatrix", &PseudoInertia::FromMatrix, bp::args("pseudo_inertia_matrix"),
+            "Returns the Pseudo Inertia from a 4x4 matrix.")
+          .staticmethod("FromMatrix")
+
+          .def(
+            "FromInertia", &PseudoInertia::FromInertia, bp::args("inertia"),
+            "Returns the Pseudo Inertia from an Inertia object.")
+          .staticmethod("FromInertia")
+
+          .def("__array__", &PseudoInertia::toMatrix)
+          .def("__array__", &__array__)
+#ifndef PINOCCHIO_PYTHON_NO_SERIALIZATION
+          .def_pickle(Pickle())
+#endif
+          ;
+        PINOCCHIO_COMPILER_DIAGNOSTIC_POP
+      }
+
+      static Scalar getMass(const PseudoInertia & self)
+      {
+        return self.mass;
+      }
+      static void setMass(PseudoInertia & self, Scalar mass)
+      {
+        self.mass = mass;
+      }
+
+      static void setH(PseudoInertia & self, const Vector3 & h)
+      {
+        self.h = h;
+      }
+
+      static Matrix3 getSigma(const PseudoInertia & self)
+      {
+        return self.sigma;
+      }
+      static void setSigma(PseudoInertia & self, const Matrix3 & sigma)
+      {
+        self.sigma = sigma;
+      }
+
+      static PseudoInertia * makeFromMatrix(const Matrix4 & pseudo_inertia_matrix)
+      {
+        return new PseudoInertia(PseudoInertia::FromMatrix(pseudo_inertia_matrix));
+      }
+
+      static void expose()
+      {
+#if PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION == 6 && EIGENPY_VERSION_AT_LEAST(2, 9, 0)
+        typedef PINOCCHIO_SHARED_PTR_HOLDER_TYPE(PseudoInertia) HolderType;
+#else
+        typedef ::boost::python::detail::not_specified HolderType;
+#endif
+        bp::class_<PseudoInertia, HolderType>(
+          "PseudoInertia",
+          "This class represents a pseudo inertia matrix and it is defined by its mass, vector "
+          "part, and 3x3 matrix part.\n\n"
+          "Supported operations ...",
+          bp::no_init)
+          .def(PseudoInertiaPythonVisitor<PseudoInertia>())
+          .def(CastVisitor<PseudoInertia>())
+          .def(CopyableVisitor<PseudoInertia>())
+          .def(PrintableVisitor<PseudoInertia>());
+      }
+
+    private:
+      static Matrix4 __array__(const PseudoInertia & self, bp::object)
+      {
+        return self.toMatrix();
+      }
+
+      struct Pickle : bp::pickle_suite
+      {
+        static boost::python::tuple getinitargs(const PseudoInertia & pi)
+        {
+          return bp::make_tuple(pi.toMatrix());
```

Comment:
getinitargs must return argument that will be provided to the constructor.

Since we have remove the constructor taking the full matrix, we will have to return mass, h and sigma.